requirements/requirements-django.txt

```diff
@@ -0,0 +1 @@
+Django>=1.8,<2.0
```

Comment:
Here's how I've been getting around that problem, while still allowing a single `pip install -r requirements/base.txt` for non-tox envs: https://github.com/edx/edx-ora2/blob/8b0670b88f215983a003d4a3e4d0b287cd3b4dcf/requirements/base.txt